modules/aws/elastikube/ign-kube-addon.tf

```diff
@@ -27,6 +27,7 @@ module "ignition_addon_coredns" {
 
   reverse_cidrs  = "${var.service_cidr}"
   cluster_dns_ip = local.cluster_dns_ip
+  replicas       = parseint(var.master_config["instance_count"], 10)
```

Comment:
instead of 10, I think it makes sense to have one? If someone is testing he/she will end up with 10 coredns pods.